src/app/pages/index/student.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs/Rx';
+import 'rxjs/add/operator/map';
+
+@Injectable()
+export class StudentService {
+
+  constructor(private httpClient: HttpClient) { }
+
+  getStudentName(studentId: string): Observable<string> {
+    const url = `http://198.211.106.128:3033/api/v1/student/${studentId}`;
```

Comment:
Maybe we should consider to add the IP to a config file instead of having it directly in the code.

Reply:
I agree, however it was out of scope for this task.